examples/va/hci/control-plane/nncp/values.yaml

```diff
@@ -172,6 +175,36 @@ data:
         gateway: 10.0.0.1
         name: subnet1
     mtu: 1500
+  swift:
+    dnsDomain: swift.example.com
+    subnets:
+      - allocationRanges:
+          - end: 172.22.0.250
+            start: 172.22.0.100
+        cidr: 172.22.0.0/24
+        gateway: 172.22.0.1
+        name: subnet1
+        vlan: 25
```

Comment:
VLAN 24 and 172.21 is the next unused range, so why VLAN 25 and 172.22 in this patch?

Because https://github.com/openstack-k8s-operators/architecture/pull/376 is using VLAN 24 and 172.21